src/openrct2/ride/RideData.cpp

```diff
@@ -454,3 +454,35 @@ TrackDrawerEntry getTrackDrawerEntry(const RideTypeDescriptor& rtd, bool isInver
 
     return descriptor.Regular;
 }
+
+int32_t RideTypeDescriptor::GetRelativeBoosterSpeed(const int32_t& absoluteSpeed) const
+{
+    auto shiftFactor = LegacyBoosterSettings.BoosterSpeedFactor;
+    if (shiftFactor == 0)
+    {
+        return absoluteSpeed;
+    }
+    if (shiftFactor > 0)
+    {
+        return (absoluteSpeed >> shiftFactor);
+    }
+    // Workaround for an issue with older compilers (GCC 6, Clang 4) which would fail the build
```

Comment:
We support neither of these any more. You can remove this comment (also from the other function). Optionally, you can also eliminate the intermediate variable `shiftFactorAbs`, but I’ll leave it to you to decide if you want that.